Neos.Flow/Documentation/TheDefinitiveGuide/PartII/Installation.rst

```diff
@@ -53,15 +53,15 @@ Directory Structure
 Let's take a look at the directory structure of a Flow application:
 
 ======================	===================================================================================
-Directory				Description
+Directory								Description
 ======================	===================================================================================
-Configuration/			Application specific configuration, grouped by contexts
-Data/					Persistent and temporary data, including caches, logs, resources and the database
-Packages/				Contains sub directories which in turn contain package directories
-Packages/Framework/		Packages which are part of the official Flow distribution
-Packages/Application/	Application specific packages
-Packages/Libraries/		3rd party libraries
-Web/					Public web root
+Configuration/					Application specific configuration, grouped by contexts
+Data/										Persistent and temporary data, including caches, logs, resources and the database
```

Comment:
Whoah, lot's of whitespace! 🤣

Reply:
Looks good in my IDE
![image](https://user-images.githubusercontent.com/307571/100470656-f23bb300-30d8-11eb-865b-884aa28b4769.png)

Reply:
Tabs vs. spaces

Reply:
Check